example/examplechecker.py

```diff
@@ -53,29 +53,29 @@ def generate_credentials(self) -> Dict[str, str]:
     def putflag(self) -> None:
         """
         This method stores a flag in the service.
-        In case multiple flags are provided, self.flag_idx gives the appropriate index.
+        In case the service has multiple flag stores, self.variant_id gives the appropriate index.
         The flag itself can be retrieved from self.flag.
         On error, raise an Eno Exception.
         :raises EnoException on error
         """
-        if self.flag_idx == 0:
+        if self.variant_id == 0:
             credentials = self.generate_credentials()
-            self.team_db[self.flag] = credentials
+            self.team_db[self.ctx] = credentials
```

Comment:
Different nomenclature between projects is not good :(

Reply:
Just a shorthand, feel free to use the long version in your checkers.

That being said, we could just have a `self.ctx_db`, `self.current_db`, or similar (even `self.chain_db`, even though that is less clear), that just gives you the thing you want?
Calling `self.team_db` with `flag` as parameter is a very common occurrence right now, and could be simplified.

Reply:
I would like something like `self.context_db` or `self.chain_db`, although in that case I would get rid of `self.ctx` as we won't have to type that a lot (or at all?) in the checker itself

Reply:
"ctx" is not at all related to "taskChainId" and therefore no valid abbreviation.

Please keep the wtfs/minute low.

Reply:
I agree that adding a separate db-shorthand is the more sensible shorthand

Reply:
thoughts on the `chain_db`?

Reply:
Not good, not terrible :)
Think we can all agree on it